src/components/login/login.tsx

```diff
@@ -1,7 +1,7 @@
 import { Container } from "react-bootstrap";
 import lsLogin from "../../assets/loginLS/ls-login.png";
 import lsRegister from "../../assets/loginLS/ls-register.png";
-import authService from "./authService";
+import authService from "../../services/auth";
 
 /** Login page */
```

Comment:
On clicking login, I get an error. is it expected?

Reply:
As discussed, this is probably because the client ID must be set in the environment, and some other provisions need to be made to simulate running under the proper domain - see documentation in Confluence.